lib/rouge/lexers/powershell.rb

```diff
@@ -14,8 +14,8 @@ class Powershell < RegexLexer
 
       # https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6
```

Comment:
For accuracy, should now change this comment to:
```
# https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_cmdletbindingattribute?view=powershell-6
```

Reply:
Have made the change (only found out that GitHub allows a direct edit and commit from the UI...).